.github/pull_request_template.md

```diff
@@ -8,15 +8,15 @@ Fixes # (issue)
 
 ## PR type
```

Comment:
Can we add somewhere before here something about the prefix/title?
e.g Please ensure your PR title is following the contribution guidelines(link)

Reply:
Added extra checkbox in checklist with link to guidelines